### DISCUSSION TITLE
TShock Forums: Shutting down on December 31, 2024 (JST)

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#3045] [Announcements]
Hey everyone.

The forums (https://tshock.co/xf/) will be shutting down on December 31, 2024 JST. We will be deleting all data. The reason for this is that as time goes on, it becomes less reasonable for us to store PII under GDPR and related agreements.

If you would like us to delay the shutdown so you can backup information, please let me know.

Importantly, if the forums go down from now to the end of the year, we will not restore service. Please backup any remaining information you wish to save before the closure date as soon as possible, to prevent this from occurring.

You have my permission to backup any and all data that you want to backup from the forums.

**Comment by Giggitybyte** [comment]
Hello, I'm trying to get into TShock plugins and I learned through old search results that the forums had a list of plugins; was a replacement for this made when the forums shut down?

**Reply by hakusaro** [reply]
there's the Pryaxis/Plugins repo, but there's kind of an ongoing stalemate as to how to handle updates moving forward; some plugin authors have effectively moved distribution to their own repos, or discord, etc. there are chinese language plugins also available on a separate resource.
